paint-timing/fcp-only/fcp-overflow.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<head>
+<title>Performance Paint Timing Test: FCP with element clipped with overflow</title>
+<style>
+    #main {
+        width: 100px;
+        height: 100px;
+        overflow: hidden;
+        display: none;
+    }
+
+    #main.contentful {
+        display: block;
+    }
+
+    #text {
+        top: 10000px;
+        position: relative;
+    }
+
+    #log {
+        display: none;
+    }
+</style>
+</head>
+<body>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../resources/utils.js"></script>
+<div id="main">
+    <div id="text">HELLO</div>
+</div>
+</body>
+
+<footer>
+<script>
+    test_fcp("First contentful paint fires even when element is hidden due to.")
```

Comment:
nit: fix description (suspense due to...)

Reply:
oops